core/shared/platform/common/math/math.c

```diff
@@ -1554,11 +1554,11 @@ atan2(double y, double x)
     return freebsd_atan2(y, x);
 }
 
-double
-sqrt(double x)
-{
-    return freebsd_sqrt(x);
-}
+// double
+// sqrt(double x) __attribute__((weak))
+// {
+//     return freebsd_sqrt(x);
+// }
```

Comment:
Could we use macro to control it? For example,
```C
#if !(defined(BH_PLATFORM_ZEPHYR) && KERNEL_VERSION_NUMBER >= xxx)
...
#endif
```
I remember that the early Zephyr version doesn't support sqrt/sqrtf yet.